griptape/tasks/base_task.py

```diff
@@ -168,3 +177,14 @@ def full_context(self) -> dict[str, Any]:
             return structure_context
         else:
             return {}
+
+    def _attach_to_structure(self, task: BaseTask):
```

Comment:
What do you think about renaming to `_add_to_structure`? "attach" conveys more of a 1-to-1 relationship.